composer.json

```diff
@@ -20,7 +20,8 @@
         "kriswallsmith/buzz": "~0.15"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.0"
+        "phpunit/phpunit": "~4.0",
+        "symfony/phpunit-bridge": "^2.7.4"
```

Comment:
PR #59 and this one are not consistent, in #59 you remove the PHPUnit dependency and in this one you keep it and add the symfony's phpunit-bridge.